portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsDateField.testcase

```diff
@@ -31,6 +31,144 @@ definition {
 		}
 	}
 
+	@description = "Verify that is able to add a future validation using the response date as parameter"
+	@priority = "5"
+	test CanAddFutureValidationUsingResponseDate {
+		property forms.functional = "true";
+
+		var displayDateDay = DateUtil.getCurrentDate();
+		var displayDateMonth = DateUtil.getCurrentMonth();
+		var displayDateYear = DateUtil.getCurrentYear();
+		var futureYear = ${displayDateYear} + 1;
+		var pastYear = ${displayDateYear} - 1;
+		var displayFutureDate = "${displayDateMonth}/${displayDateDay}/${futureYear}";
+		var displayPastDate = "${displayDateMonth}/${displayDateDay}/${pastYear}";
+		var futureHalfYear = StringUtil.extractLast("${futureYear}","0");
+
+		FormsAdmin.addForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Date");
+
+		Form.editFieldLabelMetal(fieldValue = "Date Field");
+
+		Form.gotoAdvancedTab();
+
+		FormFields.enableSwitch(fieldName = "Validation");
+
+		FormFields.validationRangeCustomDate(acceptedDateOption = "Future Dates");
+
+		FormFields.validationCustomDateStartsFrom(
+			startsFromOperation = "Plus",
+			startsFromOption = "Custom Date",
+			startsFromQuantity = "3",
+			startsFromUnit = "Days");
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormFields.viewLabel(
+			fieldLabel = "Date Field",
+			fieldName = "Date Field");
+
+		FormViewBuilder.editDate(
+			fieldName = "Date Field",
+			fieldValue = "${displayPastDate}");
+
+		FormPortlet.submitWithValidationError(validationErrorMessage = "This field is invalid.");
+
+		FormViewBuilder.editDate(
+			fieldName = "Date Field",
+			fieldValue = "${displayFutureDate}");
+
+		FormPortlet.submitSuccessfully();
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Forms");
```

Comment:
@leticiachiu, did you create a macro to speed up this process to navigate to Forms UI again without need to use the macro `ProductMenu.gotoPortlet`? If so, can you please assist @rafaellajordao here?

Reply:
here, @rafaellajordao! `FormsAdmin.openFormsAdmin(siteURLKey = "guest");`